Added option to run ADB in no-streaming mode (#1922)
beeware/briefcase: changes/1922.feature.rst

> Added to ``briefcase run android`` a ``--Xadb-install`` option for passing extra arguments to ``adb install``. This makes it possible to handle various corner cases, for example, setting the ``--no-streaming`` option to make installation work for devices running recent versions of LineageOS.

Reviewer: These notes should be written as a description of the "exciting new feature in the release", not a rehash of the commit message. 
```suggestion
The ``--Xadb-install`` option can now be used to pass arguments to ``adb install`` when running apps on Android. 
```
